Reading List/Reading List/BookDetailViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BookDetailViewController.swift
+//  Reading List
+//
+//  Created by Chris Gonzales on 1/28/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailViewController: UIViewController {
+
+    var bookController: BookController?
+    var book: Book?
+    
+    @IBOutlet weak var movieTitleTextField: UITextField!
+    @IBOutlet weak var reasonToSeeTextView: UITextView!
+    
+    @IBAction func saveTapped(_ sender: Any) {
+        guard let title = movieTitleTextField.text else {return}
+        guard let reason = reasonToSeeTextView.text else {return}
```

Comment:
Instead of doing two guard let statements you can connect them:
`guard let title = movieTitleTextField.text,
       let reason = reasonToSeeTextView.text else {return}`